### PR TITLE
Add ManhwaWeb

### PR DESCRIPTION
**Pull request by MikeZeDev**
https://github.com/manga-download/hakuneko/issues/7475

### CODE REVIEW

File: web/src/engine/websites/ManhwaWeb.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        for (let page = 1, run = true; run; page++) {
+            const { mangas, next } = await this.GetMangasFromPage(page, provider);
+            mangaList.push(...mangas);
+            run = next;
+        }
+        return mangaList;
+    }
+
+    private async GetMangasFromPage(page: number, provider: MangaPlugin): Promise<MangaResult> {
+        const { data, next } = await FetchJSON<APIMangas>(new Request(new URL(`/manhwa/library?estado=&tipo=&erotico=&demografia=&page=${page}`, this.apiUrl)));
```
Comment (ronny1982) [inline web/src/engine/websites/ManhwaWeb.ts:65]: Can't the query parameters `estado=&tipo=&erotico=&demografia=` without an explicit value be removed from the URL?

Comment (ronny1982) [inline web/src/engine/websites/ManhwaWeb.ts:65]: Is `next` necessary, wouldn't it just return an empty list on the next request which can be used as abort condition like in all the other multi-paged websites?

**Review by ronny1982 — APPROVED** [review]

File: web/src/engine/websites/ManhwaWeb.ts (hunk 1 above)
Comment (MikeZeDev) [reply web/src/engine/websites/ManhwaWeb.ts:65]: Nope. There was way more. Removing one of those i kept break the request.

Comment (MikeZeDev) [reply web/src/engine/websites/ManhwaWeb.ts:65]: Website give me a next, i use the next but yeah, sure.

**Pull request merged**